src/lib.rs

```diff
@@ -197,7 +197,7 @@ pub fn load_and_index_pois(
                     let pois =
                         LazyEs::batch_make_progress_until_value(&es_url, pois, max_batch_size)
                             .into_iter()
-                            .filter_map(|poi| poi);
+                            .flatten();
```

Comment:
This just addresses the tiny lint [`clippy::filter_map_identity`](https://rust-lang.github.io/rust-clippy/master/index.html#filter_map_identity)